packages/fxa-settings/src/images/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { ReactComponent as HeartsBroken } from './graphic_hearts_broken.svg';
+import { ReactComponent as HeartsVerified } from './graphic_hearts_verified.svg';
+import { ReactComponent as RecoveryCodes } from './graphic_recovery_codes.svg';
+import { ReactComponent as TwoFactorAuth } from './graphic_two_factor_auth.svg';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+
+import { useFtlMsgResolver } from '../models';
+
+// Use this component to add your image into the collection of images exported below.
+// If the component is not being reused across directories, it should remain inline
+// If you are introducing an image which was previously shared and had localized alt-text,
+// or a localized aria-label, please add the existing ftl id to spare translating it again.
+// in the directory where it is used in accordance with the current policy (https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/README.md#inlined-svgs)
+// Don't forget to add new FTL strings into the `en.ftl` file
+
+interface PreparedImageBaseProps {
+  className?: string;
+  Image: React.ElementType;
+}
+
+interface PreparedImageAriaHiddenProps extends PreparedImageBaseProps {
+  ariaHidden: true;
+}
+
+interface PreparedImageAriaVisibleProps extends PreparedImageBaseProps {
+  ariaHidden?: false;
+  ariaLabel: string;
+  ariaLabelFtlId: string;
+}
+
+type PreparedImageProps =
+  | PreparedImageAriaHiddenProps
+  | PreparedImageAriaVisibleProps;
+
+const PreparedImage = (props: PreparedImageProps) => {
+  const { className, ariaHidden, Image } = props;
```

Comment:
I wonder if there's ever a case where we want a shared image aria-hidden somewhere and not elsewhere. Leaving it customizable is nice though. 😎

Reply:
great question! Yeah I guess if we did we could add the `ariaHidden` prop onto the specific image component